src/test/ui/conditional-compilation/cfg_accessible-not_sure.rs

```diff
@@ -0,0 +1,85 @@
+// revisions: edition2015 edition2021
+// [edition2015]compile-flags: --edition=2015
+// [edition2021]compile-flags: --edition=2021
+
+#![feature(extern_types)]
+#![feature(cfg_accessible)]
+
+// Struct::unresolved - error
+
+struct Struct {
+    a: u8,
+    pub b: u8
+}
+
+#[cfg_accessible(Struct::a)] //~ ERROR not sure
+const A: bool = true;
+#[cfg_accessible(Struct::b)] //~ ERROR not sure
+const B: bool = true;
+#[cfg_accessible(Struct::c)] //~ ERROR not sure
+const C: bool = true;
+
+// Union::unresolved - error
+
+struct Union {
+    a: u8,
+    pub b: u8
+}
+
+#[cfg_accessible(Union::a)] //~ ERROR not sure
+const A: bool = true;
+#[cfg_accessible(Union::b)] //~ ERROR not sure
+const B: bool = true;
+#[cfg_accessible(Union::c)] //~ ERROR not sure
+const C: bool = true;
+
+// Enum::unresolved - error
+
+enum Enum {
+    A(u8),
+}
+
+#[cfg_accessible(Enum::B)] //~ ERROR not sure
```

Comment:
```suggestion
#[cfg_accessible(Enum::unresolved)] //~ ERROR not sure
```
Could you use something "obviously unresolved" for unresolved names instead of A/B/etc?